src/azure-cli-core/azure/cli/core/sdk/policies.py

```diff
@@ -103,3 +103,60 @@ def on_request(self, request):
         super().on_request(request)
         from azure.cli.core.telemetry import set_user_agent
         set_user_agent(request.http_request.headers[self._USERAGENT])
+
+
+# pylint: disable=line-too-long
+def get_custom_hook_policy(cli_ctx):
```

Comment:
Ideally, this policy should be implemented by Python SDK. Otherwise, SDK users will have to implement this policy by themselves, which is too complicated.